packages/contentstack-utilities/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/cli-utilities",
-  "version": "1.6.2",
+  "version": "1.6.3",
```

Comment:
@harshithad0703 , Please update the contentstack/package.json version. Additionally, you need to update the utilities version in each plugin. pls check with @cs-raj